usaspending_api/broker/helpers.py

```diff
@@ -0,0 +1,556 @@
+    if type == '11':
+        return 'OTHER FINANCIAL ASSISTANCE'
+    return 'UNKNOWN ASSISTANCE TYPE'
+
+
+def get_business_type_description(business_type):
+    if business_type in ('A', '00'):
+        return 'STATE GOVERNMENT'
+    if business_type in ('B', '01'):
+        return 'COUNTY GOVERNMENT'
+    if business_type in ('C', '02'):
+        return 'CITY OR TOWNSHIP GOVERNMENT'
+    if business_type in ('D', '04'):
+        return 'SPECIAL DISTRICT GOVERNMENT'
+    if business_type == 'E':
+        return 'REGIONAL ORGANIZATION'
+    if business_type == 'F':
+        return 'U.S. TERRITORY OR POSSESSION'
+    if business_type in ('G', '05'):
+        return 'INDEPENDENT SCHOOL DISTRICT'
+    if business_type in ('H', '06'):
+        return 'PUBLIC/STATE CONTROLLED INSTITUTION OF HIGHER EDUCATION'
+    if business_type == 'I':
+        return 'INDIAN/NATIVE AMERICAN TRIBAL GOVERNMENT (FEDERALLY RECOGNIZED)'
+    if business_type == 'J':
+        return 'INDIAN/NATIVE AMERICAN TRIBAL GOVERNMENT (OTHER THAN FEDERALLY RECOGNIZED)'
+    if business_type in ('K', '11'):
+        return 'INDIAN/NATIVE AMERICAN TRIBAL DESIGNATED ORGANIZATION'
+    if business_type == '12':
+        return 'OTHER NON-PROFIT'
+    if business_type == 'L':
+        return 'PUBLIC/INDIAN HOUSING AUTHORITY'
+    if business_type == 'M':
+        return 'NONPROFIT WITH 501C3 IRS STATUS (OTHER THAN INSTITUTION OF HIGHER EDUCATION)'
+    if business_type == 'N':
+        return 'NONPROFIT WITHOUT 501C3 IRS STATUS (OTHER THAN INSTITUTION OF HIGHER EDUCATION)'
+    if business_type in ('O', '20'):
+        return 'PRIVATE INSTITUTION OF HIGHER EDUCATION'
+    if business_type in ('P', '21'):
+        return 'INDIVIDUAL'
+    if business_type in ('Q', '22'):
+        return 'FOR-PROFIT ORGANIZATION (OTHER THAN SMALL BUSINESS)'
+    if business_type in ('R', '23'):
+        return 'SMALL BUSINESS'
+    if business_type == 'S':
+        return 'HISPANIC-SERVING INSTITUTION'
+    if business_type == 'T':
+        return 'HISTORICALLY BLACK COLLEGES AND UNIVERSITIES (HBCUS)'
+    if business_type == 'U':
+        return 'TRIBALLY CONTROLLED COLLEGES AND UNIVERSITIES (TCCUS)'
+    if business_type == 'V':
+        return 'ALASKA NATIVE AND NATIVE HAWAIIAN SERVING INSTITUTIONS'
+    if business_type == 'W':
+        return 'NON-DOMESTIC (NON-US) ENTITY'
+    if business_type in ('X', '25'):
+        return 'OTHER'
+    return 'UNKNOWN TYPES'
+
+
+def get_business_categories(row, data_type):
+    business_category_set = set()
+
+    if data_type == 'fabs':
+        business_types = row.get('business_types')
+        if business_types in ('R', '23'):
+            business_category_set.add('small_business')
+
+        if business_types in ('Q', '22'):
+            business_category_set.add('other_than_small_business')
+
+        if business_category_set & {'small_business', 'other_than_small_business'}:
+            business_category_set.add('category_business')
```

Comment:
not the same as the old file because it doesnt contain le.for_profit_organization == "1" or:
```
 le.for_profit_organization == "1" or
-
-            "small_business" in categories or
-            "other_than_small_business" in categories
-        ):
-            categories.append("category_business")

```

Reply:
The old method combined processing FABS and FPDS business categories. This method keep it separated. So FABS doesn't have that column available, therefore it's not checked. But it is checked below for FPDS.

Reply:
your right!